src/lib/y2storage/autoinst_profile/partition_section.rb

```diff
@@ -48,6 +48,7 @@ class PartitionSection < SectionWithAttributes
       NO_CREATE_IDS = []
       private_constant :NO_CREATE_IDS
 
+      # rubocop:disable Metrics/MethodLength
```

Comment:
Maybe moving the definition to a constant can save us from disabling the MethodLength check for `self.attributes`.

Reply:
Btw, I don't see the check being restored for the rest of the file.